suite-native/assets/src/components/AssetItem.tsx

```diff
@@ -72,6 +75,17 @@ export const AssetItem = memo(
             ) => selectNumberOfUniqueTokensForCoinPerDevice(state, cryptoCurrencySymbol),
         );
 
+        const accountsWithStaking = useSelector(
```

Comment:
This will break perf of this component and will rerender with every change to any account which is happening all the time. Please memoize this selector or better just return that boolean we need here directly from selector.